include/iso15118/message/schedule_exchange.hpp

```diff
@@ -112,7 +106,7 @@ struct ScheduleExchangeResponse {
 
     struct PriceRuleStack {
         uint32_t duration;
-        std::array<PriceRule, PRICE_RULE_LENGTH> price_rule;
+        std::vector<PriceRule> price_rule;
```

Comment:
Because I need the actual number of `price_rule` elements for the conversion to cb exi struct/array. With `std::array` I get with `size()` the maximum length of the defined array but not how many elements are actually set.
For example: The ScheduleExchange state definies a price_rule array with 2 PriceRule elements. With `std::array` I dont know how to set the prober length of the array in the cb exi array definition for price_rule. With `size()` I get `PRICE_RULE_LENGTH` and not 2.

Reply:
right, well that's unfortunate with `std::array` :(

Reply:
I would like to use `std::array` but in this case `std::vector` is better suited